api/src/test/java/org/pmiops/workbench/api/WorkspacesControllerTest.java

```diff
@@ -4,13 +4,10 @@
 import static com.google.common.truth.Truth.assertWithMessage;
 import static junit.framework.TestCase.assertEquals;
 import static junit.framework.TestCase.fail;
+import static org.mockito.Matchers.any;
 import static org.mockito.Matchers.anyListOf;
 import static org.mockito.Matchers.anyString;
-import static org.mockito.Mockito.doThrow;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.never;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.*;
```

Comment:
revert *

Reply:
Doing. I don't know when or how my settings keep getting reverted, because I know for a fact last time you commented this, I changed my settings to never do this...